packit_service/worker/handlers/abstract.py

```diff
@@ -25,32 +25,80 @@
 """
 import logging
 import shutil
+from collections import defaultdict
 from os import getenv
 from pathlib import Path
-from typing import Dict, Optional, Type, List
+from typing import Dict, Optional, Type, List, Set
 
 from packit.api import PackitAPI
 from packit.config import JobConfig, JobType
 from packit.local_project import LocalProject
 
 from packit_service.config import ServiceConfig
+from packit_service.sentry_integration import push_scope_to_sentry
 from packit_service.service.events import Event, TheJobTriggerType
 from packit_service.worker.result import HandlerResults
-from packit_service.sentry_integration import push_scope_to_sentry
 
 logger = logging.getLogger(__name__)
 
-JOB_NAME_HANDLER_MAPPING: Dict[JobType, Type["JobHandler"]] = {}
+JOB_REQUIRED_MAPPING: Dict[JobType, Set[Type["JobHandler"]]] = defaultdict(set)
```

Comment:
Just wondering why this is named differently from the other `MAP_SOMETHING_TO_SOMETHING`, but I don't mind.